packages/transmute-cli/package.json

```diff
@@ -3,8 +3,7 @@
   "version": "0.2.2",
   "description": "transmute cli",
   "scripts": {
-    "build": "babel src -d lib",
```

Comment:
So we are no longer going to be packaging this?

Reply:
its not necessary to use babel to build a commonjs cli. i removed this because it makes things more simple.